front/app/containers/LandingPage/SignUpButton.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Button from 'components/UI/Button';
+import styled from 'styled-components';
+import { media } from 'utils/styleUtils';
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from './messages';
+
+const CTAButton = styled(Button).attrs(() => ({
+  fontWeight: '500',
+  padding: '13px 22px',
+  buttonStyle: 'primary-inverse',
+  className: 'e2e-signed-out-header-cta-button',
+}))`
+  margin-top: 38px;
+
+  ${media.smallerThanMinTablet`
+    margin-top: 30px;
+  `}
+`;
+
+interface Props {
+  signUpIn: (event) => void;
```

Comment:
@alexander-cit I saw your change to MouseEvent, but it can also be a keyboard event (tab + enter):

```
import React, { MouseEvent, KeyboardEvent } from 'react';

interface Props {
  signUpIn: (event: MouseEvent | KeyboardEvent) => void;
}
```

Reply:
It works! Thank you! Before I couldn't figure out where I can get these events classes.